plugins/modules/vmware_dvswitch.py

```diff
@@ -702,11 +720,16 @@ def check_netFlow_config(self):
                 changed = changed_internalFlowsOnly = True
                 internalFlowsOnly_previous = current_config.internalFlowsOnly
                 new_config.internalFlowsOnly = self.netFlow_internal_flows_only
+            if current_switchIpAddress != self.netFlow_switch_ip:
```

Comment:
```suggestion
            if self.netFlow_switch_ip is not None and current_switchIpAddress != self.netFlow_switch_ip:
```

Sorry, I didn't find the time to really test this. Well, not until today and I've found this issue.

When I tested with a switch that already had an IP configured, the module always reported a change (because the configured IP isn't `None`) without changing anything. I guess the API ignored this. (Not sure how to unset the IP, maybe explicitly give the empty string `""` 🤷)

Anyway, this would be quite annoying for existing users. I think you should only test this if the parameter is set to avoid this.

Reply:
Looks like I stuck too close to the path of the other variables, yeah it totally makes sense to ignore `None` and only unset by empty string 👍🏻

Reply:
Looks like you're fine with my proposal. Please don't merge it yourself, there's another minor issue I'd like to fix and would do both in one go.

Reply:
It took me too long to check if this isn't introducing any side effects anyways, thank you for already merging it 🙂

Reply:
I don't think this would have any side effects. If it turns out I'm wrong, I hope I'm faster to fix this than I've been here.